gtkm/stats/github_fetcher.py

```diff
@@ -1,8 +1,108 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Cookie
+from fastapi.responses import JSONResponse
+from typing import Optional
+import json as JSON
+import httpx
```

Comment:
`ModuleNotFoundError: No module named 'httpx'`  

To nie jest biblioteka standardowa.  
Powinna być dodana do requirements